### PR TITLE
Icon tweak and text changes

### PR DESCRIPTION
**Pull request by DeeJayLSP**
The icon has been slightly tweaked to play nicer with outlines and have a smoother gradient.

Before:
image
After:
image

Before (outlined):
image
After (outlined):
image

The font used for "Blazium" is **Roboto**, with character spacing reduced to -4.0 (in GIMP and Inkscape terms)

The font used for "Game Engine" is **Poppins**.

Most of these changes were based on feedbacks by the team in Discord.

The PNG optimization tool caught a few files, so they ended up being included.

### CODE REVIEW

**Review by sshiiden — APPROVED** [review]

**Review by Norrox — APPROVED** [review]

File: platform/android/java/lib/res/mipmap-xxxhdpi/icon_foreground.png
Comment (jss2a98aj) [inline platform/android/java/lib/res/mipmap-xxxhdpi/icon_foreground.png:1]: This replacement appears to have the old and new version layered on top of each other, at least in the web preview

**Review by jss2a98aj — CHANGES_REQUESTED** [review]
I quite like the changes made here. Nice job spotting the previously unchanged VR splash. I only spotted one potential issue

File: platform/android/java/lib/res/mipmap-xxxhdpi/icon_foreground.png
Comment (Norrox) [reply platform/android/java/lib/res/mipmap-xxxhdpi/icon_foreground.png:1]: lol, good catch i didn't even see that

Comment (DeeJayLSP) [reply platform/android/java/lib/res/mipmap-xxxhdpi/icon_foreground.png:1]: Fixed

**Review by jss2a98aj — DISMISSED** [review]

**Comment by Starkium** [comment]
going to need to compare with the other branch that had the revised splash

**Comment by DeeJayLSP** [comment]
Rebased

**Comment by DeeJayLSP** [comment]
I have spotted a few oversights in the icon that makes it go out of bounds if placed in a circle. I'll need to rework this PR a bit.

**Comment by DeeJayLSP** [comment]
Rework finished.

**Review by Norrox — APPROVED** [review]

**Comment by Norrox** [comment]
> going to need to compare with the other branch that had the revised splash

A little smaller and white text, for me it's fine whichever we choose 
image

**Review by jss2a98aj — DISMISSED** [review]
Nothing stood out as wrong, and the ico files had the required layers. The recent tweak also seems fine

**Review by RandomOfNoWhere — APPROVED** [review]

**Comment by Starkium** [comment]
> > going to need to compare with the other branch that had the revised splash
> 
> A little smaller and white text, for me it's fine whichever we choose image

maybe we should scale it up? it seems in the other branch they had intentionally scaled it up there

File: misc/dist/ios_xcode/godot_ios/Images.xcassets/SplashImage.imageset/splash.png
Comment (jss2a98aj) [inline misc/dist/ios_xcode/godot_ios/Images.xcassets/SplashImage.imageset/splash.png:1]: Was this file supposed to be added?

Comment (DeeJayLSP) [reply misc/dist/ios_xcode/godot_ios/Images.xcassets/SplashImage.imageset/splash.png:1]: I was dropping the splash there, then using a command to overwrite the others. I ended up forgetting to delete it this time.

**Review by jss2a98aj — APPROVED** [review]

**Review by Bioblaze — APPROVED** [review]

**Review by sshiiden — APPROVED** [review]

**Pull request merged**